Rotator/MinCountMaxStorageSizeRotator.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Zenstruck\BackupBundle\Rotator;
+
+use Zenstruck\BackupBundle\Destination\Backup;
+use Zenstruck\BackupBundle\Utils\Filesize;
+
+final class MinCountMaxStorageSizeRotator
```

Comment:
New implementation of rotator -> max size but keep minimum number of backups regardless of size.